packages/components/PersonaCoin/package.json

```diff
@@ -23,14 +23,14 @@
     "prettier-fix": "fluentui-scripts prettier --fix true"
   },
   "dependencies": {
-    "@uifabricshared/foundation-compose": "^1.9.0",
-    "@fluentui-react-native/adapters": ">=0.6.0 <1.0.0",
-    "@fluentui-react-native/theme-tokens": ">=0.1.0 <1.0.0",
-    "@fluentui-react-native/tokens": ">=0.9.0 <1.0.0",
-    "@uifabricshared/foundation-composable": ">=0.8.1 <1.0.0",
-    "@uifabricshared/foundation-settings": ">=0.9.0 <1.0.0",
-    "@uifabricshared/foundation-tokens": ">=0.9.0 <1.0.0",
-    "@uifabricshared/theming-ramp": ">=0.14.0 <1.0.0"
+    "@uifabricshared/foundation-compose": "^1.9.2",
+    "@fluentui-react-native/adapters": ">=0.6.1 <1.0.0",
+    "@fluentui-react-native/theme-tokens": ">=0.2.1 <1.0.0",
+    "@fluentui-react-native/tokens": ">=0.9.2 <1.0.0",
+    "@uifabricshared/foundation-composable": ">=0.8.2 <1.0.0",
+    "@uifabricshared/foundation-settings": ">=0.9.1 <1.0.0",
+    "@uifabricshared/foundation-tokens": ">=0.9.1 <1.0.0",
+    "@uifabricshared/theming-ramp": ">=0.14.2 <1.0.0"
```

Comment:
Are these updates a part of the PR? Probably fine but I thought we mostly let beachball handle that.

Reply:
I'm pretty sure these changes came in from a merge from master... The only new line from the PR should be adding theme-tokens.